manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml

```diff
@@ -0,0 +1,28 @@
+PackageIdentifier: OpenStreetMap.Josm
+Publisher: OpenStreetMap
+PackageName: JOSM
+PackageVersion: 17833
```

Comment:
Package version need to be a string according to the manifest singleton JSON schema.
https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json